demo/executor/src/lib.rs

```diff
@@ -239,7 +239,7 @@ mod tests {
 			UncheckedExtrinsic::new(extrinsic, signature)
 		}).collect::<Vec<_>>();
 
-		let extrinsics_root = ordered_trie_root(extrinsics.iter().map(Encode::encode)).0.into();
+		let extrinsics_root = ordered_trie_root::<KeccakHasher, _, _>(extrinsics.iter().map(Encode::encode)).0.into();
```

Comment:
Shouldn't this also define the `NodeCodec`?

Reply:
Ideally yes, but `triehash` is intrinsically tied to RLP and untying that amounts to, I believe, a rewrite. [Issue](https://github.com/paritytech/parity-common/issues/50).